lib/runners/processor/haml_lint.rb

```diff
@@ -26,34 +28,11 @@ class Processor::HamlLint < Processor
       )
     end
 
-    # DEPRECATED: Implicit dependencies
     # @see https://github.com/sider/runner_rubocop/blob/3.1.0/lib/entrypoint.rb#L27-L52
     OPTIONAL_GEMS = [
-      GemInstaller::Spec.new(name: "meowcop", version: []),
-      GemInstaller::Spec.new(name: "onkcop", version: []),
-      GemInstaller::Spec.new(name: "deka_eiwakun", version: []),
-      GemInstaller::Spec.new(name: "forkwell_cop", version: []),
-      GemInstaller::Spec.new(name: "cookstyle", version: []),
-      GemInstaller::Spec.new(name: "rubocop-rails_config", version: []),
-      GemInstaller::Spec.new(name: "salsify_rubocop", version: []),
-      GemInstaller::Spec.new(name: "otacop", version: []),
-      GemInstaller::Spec.new(name: "unasukecop", version: []),
-      GemInstaller::Spec.new(name: "sanelint", version: []),
-      GemInstaller::Spec.new(name: "hint-rubocop_style", version: []),
-      GemInstaller::Spec.new(name: "rubocop-salemove", version: []),
-      GemInstaller::Spec.new(name: "mad_rubocop", version: []),
-      GemInstaller::Spec.new(name: "unifacop", version: []),
-      GemInstaller::Spec.new(name: "ws-style", version: []),
-      GemInstaller::Spec.new(name: "rubocop-config-umbrellio", version: []),
-      GemInstaller::Spec.new(name: "pulis", version: []),
-      GemInstaller::Spec.new(name: "gc_ruboconfig", version: []),
-      GemInstaller::Spec.new(name: "fincop", version: []),
-      GemInstaller::Spec.new(name: "rubocop-github", version: []),
-      GemInstaller::Spec.new(name: "ezcater_rubocop", version: []),
-      GemInstaller::Spec.new(name: "rubocop-rspec", version: []),
-      GemInstaller::Spec.new(name: "rubocop-cask", version: []),
-      GemInstaller::Spec.new(name: "rubocop-thread_safety", version: []),
-    ].freeze
+      *Processor::RuboCop::OPTIONAL_GEMS,
+      # additional gems for HAML-Lint
+    ]
```

Comment:
[nitpick] For consistency and safety.

```suggestion
    ].freeze
```